scanner/pkg/client/options.go

```diff
@@ -4,53 +4,117 @@ import (
 	"github.com/stackrox/rox/pkg/mtls"
 )
 
+var defaultConnOptions = connOptions{
+	mTLSSubject: mtls.ScannerSubject,
+	address:     ":8443",
+	serverName:  "scanner-v4.stackrox",
+	skipTLS:     false,
+}
+
 var defaultOptions = options{
-	mtlsSubject:   mtls.ScannerSubject,
-	address:       ":8443",
-	serverName:    "scanner-v4.stackrox",
-	withTLSVerify: true,
+	indexerOpts: defaultConnOptions,
+	matcherOpts: defaultConnOptions,
+	comboMode:   true,
```

Comment:
doesn't seem like this default value is ever used. Should we just remove it?

Reply:
It reflects the state of the previous values (indexerOpts and matcherOpts), since they are the same, `comboMode` should be true.

Reply:
If you don't have any other comments, could you please resolve the thread?